staging/src/k8s.io/client-go/plugin/pkg/client/auth/azure/azure_test.go

```diff
@@ -223,14 +221,111 @@ func TestAzureTokenSource(t *testing.T) {
 	}
 }
 
+// func fakeRefreshToken()
+func TestAzureTokenSourceRetrieveFromCfg(t *testing.T) {
```

Comment:
about test case code style, could you refer to https://github.com/kubernetes/kubernetes/blob/3a828af9630f1bee69a1d69a47e761d6880f5cf2/staging/src/k8s.io/legacy-cloud-providers/azure/azure_config_test.go#L91